model/postgres/postgres_resource.rb

```diff
@@ -95,6 +95,10 @@ def set_firewall_rules
     private_subnet.firewalls.first.replace_firewall_rules(vm_firewall_rules)
   end
 
+  def ca_certificates
+    [root_cert_1, root_cert_2].join("\n") if root_cert_1 && root_cert_2
```

Comment:
We do similar operation in PostgresServerNexus as well. Could you please update it to use this function?

Reply:
Done